### PR TITLE
[3.3] Backport of 2 small fixes

### PR DESCRIPTION
**Pull request by FdaSilvaYY**
Partial Backport of #24968 to stable branches.

3 changed lines.

##### Checklist

- [ ] documentation is added or updated
- [ ] tests are added or updated

### CODE REVIEW

File: apps/lib/s_cb.c [hunk 1]
```diff
@@ -746,6 +746,7 @@ static STRINT_PAIR tlsext_types[] = {
     {"psk kex modes", TLSEXT_TYPE_psk_kex_modes},
     {"certificate authorities", TLSEXT_TYPE_certificate_authorities},
     {"post handshake auth", TLSEXT_TYPE_post_handshake_auth},
+    {"early_data", TLSEXT_TYPE_early_data},
```
Comment (tom-cosgrove-arm) [inline apps/lib/s_cb.c:749]: There is a trailing TAB on this line - please remove it

**Comment by tom-cosgrove-arm** [comment]
Which branches do we want this on? It applies cleanly all the way back to 3.0, but the discussions in #24968 only talk about 3.3.

**Comment by t8m** [comment]
IMO should be applied all the way through to 3.0

**Comment by tom-cosgrove-arm** [comment]
ping @FdaSilvaYY to fix the whitespace error

**Comment by FdaSilvaYY** [comment]
Whitespace fixed !

**Review by tom-cosgrove-arm — APPROVED** [review]
LGTM - thanks

**Review by t8m — APPROVED** [review]

**Review by paulidale — APPROVED** [review]

**Comment by t8m** [comment]
Merged to the 3.3, 3.2, 3.1 and 3.0 branches. Thank you for your contribution.

**Pull request closed without merging**